Content.Server/Paper/PaperSystem.cs

```diff
@@ -160,6 +160,9 @@ private void OnInputTextMessage(EntityUid uid, PaperComponent paperComp, PaperIn
                 if (args.Session.AttachedEntity != null)
                     _adminLogger.Add(LogType.Chat, LogImpact.Low,
                         $"{ToPrettyString(args.Session.AttachedEntity.Value):player} has written on {ToPrettyString(uid):entity} the following text: {args.Text}");
+
+                if (paperComp.Sound != null)
+                    _audio.PlayPvs(paperComp.Sound, uid);
```

Comment:
```suggestion
                _audio.PlayPvs(paperComp.Sound, uid);
```
don't need to check for null SoundSpecifier.